app/controllers/admin/volunteers.py

```diff
@@ -217,7 +217,7 @@ def deleteBackgroundCheck():
 
 @admin_bp.route('/updateProgramManager', methods=["POST"])
 def updateProgramManager():
-    if g.current_user.isCeltsAdmin:
+    if g.current_user.isCeltsAdmin or g.current_user.isProgramManager():
```

Comment:
This logic would let student program managers add or remove other program managers to programs. This should not be allowed for student program managers. This should be allowed only for admins